docs/_writing_parameters.rst

```diff
@@ -77,7 +77,7 @@ support `Cheetah <http://www.cheetahtemplate.org/docs/users_guide_html/users_gui
 ::
 
     #if $mask_regions
-    -M $mask_regions
+    -M "$mask_regions"
```

Comment:
Less risky with regards to special characters - a relatively recent tweak to the best practice guidelines which used to use double quotes: https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html#command-formatting https://github.com/galaxy-iuc/standards/commit/25a1a656abea799cd7d3c612201fa53892d97334